src/main/java/io/opentracing/contrib/global/ServiceLoader.java

```diff
@@ -0,0 +1,89 @@
+package io.opentracing.contrib.global;
+
+import javax.imageio.spi.ServiceRegistry;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Iterator;
+import java.util.concurrent.Callable;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Loader class to delegate to JDK 6 ServiceLoader or fallback to the old {@link ServiceRegistry} (only for java 1.5).
```

Comment:
I'm not sure OpenTracing compiles to java 5.. does it? If not please axe as there's a lot of code to review